### PR TITLE
Add render test for collator expressions.

### PR DESCRIPTION
**Pull request by ChrisLoer**
The expression test suite is a more natural home for collator tests, but we don't have the expression test suite hooked up on Android and the collator expression has Android-specific functionality that needs testing.

My plan is to cover collator expressions using the Android render test suite that @tobrun just implemented.

/cc @anandthakker @tobrun @LukasPaczos

### CODE REVIEW

File: test/integration/render-tests/collator/default/style.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                    ["==", ["var", "op"], "<"],
+                    ["<", ["var", "a"], ["var", "b"], ["var", "collation"]],
+                    ["==", ["var", "op"], "<="],
+                    ["<=", ["var", "a"], ["var", "b"], ["var", "collation"]],
+                    ["==", ["var", "op"], ">"],
+                    [">", ["var", "a"], ["var", "b"], ["var", "collation"]],
+                    ["==", ["var", "op"], ">="],
+                    [">=", ["var", "a"], ["var", "b"], ["var", "collation"]],
+                    false
+                ]]
```
Comment (anandthakker) [inline test/integration/render-tests/collator/default/style.json]: This is a nice test expression! :)

Nit: could it be a little simpler with `match`?

```js
                ["to-string", ["match",
                    ["var", "op"],
                    "==", ["==", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    "!=", ["!=", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    "<", ["<", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    "<=", ["<=", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    ">", [">", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    ">=", [">=", ["var", "a"], ["var", "b"], ["var", "collation"]],
                    false
                ]]
```

File: test/integration/render-tests/collator/default/style.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 512,
```
Comment (anandthakker) [inline test/integration/render-tests/collator/default/style.json]: Can this work at a smaller size (w/ a corresponding smaller font size)?  Or does that make the image illegible?

File: test/integration/render-tests/collator/default/style.json [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+          },
+          {
+            "type": "Feature",
+            "properties": {
+                "a": "a",
+                "b": "ä",
+                "operator": "<",
+                "case-sensitive": true,
+                "diacritic-sensitive": false
+            },
```
Comment (anandthakker) [inline test/integration/render-tests/collator/default/style.json]: This combination looks like a duplicate of the one at line 67

File: test/integration/render-tests/collator/default/style.json (hunk 1 above)
Comment (ChrisLoer) [reply test/integration/render-tests/collator/default/style.json]: Good idea! Yeah it was actually nice to be forced to think a little bit about a non-trivial collator expression.

File: test/integration/render-tests/collator/default/style.json (hunk 3 above)
Comment (ChrisLoer) [reply test/integration/render-tests/collator/default/style.json]: Oops, meant that to be diacritic-sensitive!

File: test/integration/render-tests/collator/default/style.json (hunk 2 above)
Comment (ChrisLoer) [reply test/integration/render-tests/collator/default/style.json]: Yeah I brought it down to 256x256 at 11 point font, still easy to read.

**Pull request merged**